lib/hanami/cli/base_command.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "dry/inflector"
+require "hanami/cli/command"
+require "hanami/utils/files"
+
+module Hanami
+  class CLI
+    # This should really be named `Command`, but I'm temporarily using a
+    # different name to make it easier to drop this in without having to mess
+    # with existing code
+    class BaseCommand < Hanami::CLI::Command
+      attr_reader :out
+      attr_reader :inflector
+      attr_reader :files
+
+      # TODO: consider injecting `out, files` from level of Snowpack::CLI?
+      def initialize(
+        command_name:,
+        out: $stdout,
+        inflector: Dry::Inflector.new,
```

Comment:
@jodosha Yep, absolutely we should be passing the same inflector as configured for the app.

What you're seeing here is just the default param for initialising command classes _before_ they've been provided the app object.

In actual usage, this won't be used, because with the way I've set up the CLI, the `#with_application` method is called to "prepare" each already-initialized command to be used with the application. And I've just pushed https://github.com/hanami/hanami/pull/1019/commits/b075fddf25cf5a075fb17fa65c3cb2efffeec9f8 which ensures the application's inflector is passed through at that point too.